gsdk/Cargo.toml

```diff
@@ -30,5 +30,11 @@ sp-core.workspace = true
 # deps for features `testing`
 rand = { workspace = true, optional = true }
 
+[dev-dependencies]
+rand = { workspace = true }
+demo-waiter.workspace = true
```

Comment:
Can we use `= { workspace = true }` everywhere instead of `.workspace = true`? Even in this single file there are 2 different ways of it

Reply:
There is a short `dep.workspace = true` entry in the Rust book: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace

But I vote too for the consistent style across the whole manifest.

Reply:
I would go with more unified approach. Moreover, my Better TOML does not like the shortend version :)

Reply:
Let's use `<dep>.workspace = true` everywhere.